Containerfile

```diff
@@ -4,6 +4,10 @@ LABEL maintainer="LSIT Systems <lsitops@ucsb.edu>"
 
 USER root
 
+RUN apt-get update && \
+    apt-get install -y texlive-full lmodern libbz2-dev nano && \
```

Comment:
Here's the test we added for pstat215 for lmodern.sty.  Since you used `&&` instead of `;` here, this should be sane, but there was a case in the other image once where it managed to get past this step without lmodern being actually installed in the image.

https://github.com/UCSB-PSTAT/pstat215-container-image/blob/main/Jenkinsfile#L23

Reply:
added